homeassistant/components/dremel_3d_printer/config_flow.py

```diff
@@ -42,7 +42,8 @@ async def async_step_user(
             api = await self.hass.async_add_executor_job(Dremel3DPrinter, host)
         except (ConnectTimeout, HTTPError, JSONDecodeError):
             errors = {"base": "cannot_connect"}
-        except Exception:  # pylint: disable=broad-except
+        except Exception as ex:  # pylint: disable=broad-except
+            LOGGER.exception(ex)
```

Comment:
You don't pass the exception to `LOGGER.exception`, you pass a message and it will find the exception because it's called from an except block.